core/new-gui/src/app/workspace/service/drag-drop/drag-drop.service.ts

```diff
@@ -228,6 +259,80 @@ export class DragDropService {
         y: ui.offset.top
       }
     });
+
+    if (this.suggestionOperator !== undefined) {
+      const operatorID = this.suggestionOperator.operatorID;
+      const status = true;
+      this.operatorSuggestionUnhighlightStream.next({status, operatorID});
+      this.suggestionOperator = undefined;
+    }
+  }
+
+  private handleMouseMove(): void {
+    let isDone = false;
+
+    Observable.fromEvent<MouseEvent>(window, 'mouseup').first()
+      .subscribe(
+        () => isDone = true,
+        (error) => console.error(error)
+      );
+
+    Observable.fromEvent<MouseEvent>(window, 'mousemove').auditTime(100).map(
+      value => [value.clientX, value.clientY]
+    ).subscribe(
+      (value) => {
+        if (!isDone) {
```

Comment:
rather than using this if-statement here, use `filter` method.

Reply:
In addition, rename `value` to `mouseCoordinates`

Reply:
fix it